libs/sdk-ui-kit/src/Dialog/ConversionTouchPointDialogBase.tsx

```diff
@@ -0,0 +1,130 @@
+// (C) 2021 GoodData Corporation
+import React, { useEffect, useState } from "react";
+import { useIntl } from "react-intl";
+import { useHubspotForm } from "@aaronhayes/react-use-hubspot-form";
+import { LoadingComponent } from "@gooddata/sdk-ui";
+
+import { Button } from "../Button";
+import { HubSpotFormLocale } from "@aaronhayes/react-use-hubspot-form/build/main/hubspot-form-locale.enum";
```

Comment:
- Just a confirmation: the import path looks like an internal path in `@aaronhayes/react-use-hubspot-form` package. Can we use a more standard path to import `HubSpotFormLocale`?
- Also, should we move this import line up to line 5 above, so that imports related to `@aaronhayes/react-use-hubspot-form` can be placed next to each other?